spikesorters/basesorter.py

```diff
@@ -48,7 +47,7 @@ class BaseSorter:
     def __init__(self, recording=None, output_folder=None, verbose=False,
                  grouping_property=None, delete_output_folder=False):
 
-        assert self.installed, """This sorter {} is not installed.
+        assert self.is_installed(), """This sorter {} is not installed.
```

Comment:
@samuelgarcia can you change this to "The sorter ..."?

Reply:
And run again tests on travis ? OK